src/Foundatio.RabbitMQ/Messaging/RabbitMQMessageBus.cs

```diff
@@ -117,21 +131,22 @@ public class RabbitMQMessageBus : MessageBusBase, IMessageBus {
                 return;
             _logger.Trace("Message Publish: {messageType}", messageType.FullName);
 
-            // RabbitMQ only supports delayed messages with a third party plugin.
-            if (delay.HasValue && delay.Value > TimeSpan.Zero) {
```

Comment:
Well this is the only unit test case which is failing for me. I am still trying to understand its behavior.

Test Name:	CanSendDelayedMessage
Test FullName:	Foundatio.RabbitMQ.Tests.Messaging.RabbitMqMessageBusTests.CanSendDelayedMessage
Test Source:	C:\Users\barkha.shah\Foundatio\test\Foundatio.RabbitMQ.Tests\Messaging\RabbitMqMessageBusTests.cs : line 39
Test Outcome:	Failed
Test Duration:	0:10:34.256

Result StackTrace:	
at Xunit.Assert.Equal[T](T expected, T actual, IEqualityComparer`1 comparer)
   at Foundatio.Tests.Messaging.MessageBusTestBase.<CanSendDelayedMessage>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Xunit.Sdk.TestInvoker`1.<>c__DisplayClass46_1.<<InvokeTestMethodAsync>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Xunit.Sdk.ExecutionTimer.<AggregateAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Xunit.Sdk.ExceptionAggregator.<RunAsync>d__9.MoveNext()
Result Message:	
Assert.Equal() Failure
Expected: 0
Actual:   998